# Request fields
int8       interaction_id
---
# Response fields
bool success
